ibm/service/scc/resource_ibm_scc_scope.go

```diff
@@ -126,20 +70,15 @@ func ResourceIbmSccScope() *schema.Resource {
 				Description: "A list of scopes/targets to exclude from a scope.",
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
-						"account_id": &schema.Schema{
+						"scope_type": &schema.Schema{
```

Comment:
same here

Reply:
see https://github.com/IBM-Cloud/terraform-provider-ibm/pull/5891#discussion_r1966324814

Reply:
@pavanm87 can you elaborate why defining`scope_type` is incorrect if `scope_type` is being returned from the response? https://github.com/IBM-Cloud/terraform-provider-ibm/pull/5891#discussion_r1966329729